### DISCUSSION TITLE
2  inputs channels(CT scans) to re-train nnUnet

### DISCUSSION THREAD

**Discussion by agatadefr** [MIC-DKFZ/nnUNet#591] [Q&A] [answered]
Hello,

I am working on the liver and lesion segmentation. I saw that there are pretrained models available but there are trained on only one scan.  I have a set of my dataset.  I would like to run and train nnUnet on 2 CT scans (arterial and portal face). Inputs to train :

1. CT artierial + arterial_labels 
2. CT portal + portal_labels

Could you advice something ? What is the easiest way to do that ?

I would appreciate any advice.

**Comment by FabianIsensee** [comment, marked as answer]
Hi, there is unfortunately no pretrained model that can handle these inputs. You would need to train a new model on your data
Best,
Fabian

**Reply by agatadefr** [reply]
@FabianIsensee  thanks for your reply. 
I have one concern about defining input dataset in json file.

in imagesTr : I have 2 CT images : CT_arterial and CT_portal. I can declare images like this : 

- liver_01_0000.nii.gz
- liver_01_0001.nii.gz

And about labelsTr:
I have different labels to liver_01_0000.nii.gz and liver_01_0001.nii.gz ? 
How should I define or declare in json file ? 
liver_01 .... ???
or it should be only one label file ?

**Reply by FabianIsensee** [reply]
Hi, 
it is probably best if you just look at an example dataset, for example Task05 from the medical segmentation decathlon (after `nnUNet_convert_decathlon_task`).
TLDR: In your case you need two images per case in imagesTr: liver_01_0000.nii.gz, liver_01_0001.nii.gz 
and one label per case in labelsTr: liver_01.nii.gz
These three images need to have the same geometry (shape, direction, spacing, origin). Labels in liver_01.nii.gz need to be consecutive integers (0, 1, 2, 3, 4, ..., num_classes) with 0 being background.
Best,
Fabian

**Reply by agatadefr** [reply]
Hi @FabianIsensee,

@ Thanks for your suggestion. I follow it and test. It seems to work in our case. I gonna follow that path.
I am really grateful for your help. 
Best,
Agata
